Jenkinsfile

```diff
@@ -29,17 +29,11 @@ pipeline {
 
     stage('Re-install NeMo') {
       steps {
-        sh './reinstall.sh'
+        sh './reinstall.sh && pip install -U tqdm'
```

Comment:
install -U tqdm doesnt look nice, the current candidate added a version constraint on tqdm which will solve this

Reply:
Added that one but did not solved it. It was weird! I removed it and now it is working. I am not sure what has happens, but removed this one.